maxim/logger/livekit/store.py

```
@@ -1,9 +1,11 @@
 import time
 import weakref
+from dataclasses import dataclass
 from datetime import datetime
 from enum import Enum
+from io import BytesIO
 from threading import Lock
-from typing import Callable, Optional, TypedDict, Union
+from typing import Any, Callable, Optional, TypedDict, Union
```

Ruff (0.11.9)

`typing.TypedDict` imported but unused

Remove unused import: `typing.TypedDict`

(F401)